lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -125,6 +125,23 @@ mpl.figure.prototype._init_canvas = function() {
     // Throttle sequential mouse events to 1 every 20ms.
     rubberband.mousemove('motion_notify', mouse_event_fn);
 
+    function scroll_fn(event) {
+        event['data'] = 'scroll'
+        // http://www.adomas.org/javascript-mouse-wheel/
+        if (event.wheelDelta) { /* IE/Opera. */
+                event.step = event.wheelDelta/120;
+        } else if (event.detail) { /** Mozilla case. */
+            /** In Mozilla, sign of delta is different than in IE.
+             * Also, delta is multiple of 3.
+             */
+            event.step = -event.detail/3;
+        }
+        mouse_event_fn(event);
+    }
+
+    canvas_div[0].addEventListener('DOMMouseScroll', scroll_fn, false);
+    canvas_div[0].addEventListener('mousewheel', scroll_fn, false);
```

Comment:
My current theory is that the other functions are jquery sourced methods (and `canvas_div` is a jquery object) where as `addEventListener` is a native javastcript function which needs to be called on the actual html element (?) which is the first object returned when you index into a jquery object.  I _think_ we should be using `canvas_div.scroll(scroll_fun)` (https://api.jquery.com/scroll/)  here (might have to fake up the `canval_div` to be 'scrollable' which hopefully we can do without making the browser try to scroll it.